gk/main.c

```diff
@@ -180,10 +182,18 @@ extract_packet_info(struct rte_mbuf *pkt, struct ipacket *packet)
 	return ret;
 }
 
+static inline uint64_t
+calc_request_expire_at(uint64_t now)
+{
+	return now + REQ_EXPIRATION_SEC * picosec_per_cycle / 1000000000000L;
```

Comment:
The return should be in the unit of ``cycles``. 1 second is ``10^12`` pico-seconds. Shouldn't the return be ``now + REQ_EXPIRATION_SEC * 1000000000000L / picosec_per_cycle``?